composer.json

```
@@ -18,10 +18,10 @@
     "symfony/config": "^3.4 || ^4.0 || ^5.0 || ^6.0",
     "symfony/console": "^3.4 || ^4.0 || ^5.0 || ^6.0",
     "symfony/dependency-injection": "^3.4 || ^4.0 || ^5.0 || ^6.0",
+    "symfony/filesystem": "^5.0",
```

I think this should be `^5.4` as the `Path` class was added very recently: https://symfony.com/blog/new-in-symfony-5-4-filesystem-path-class

```suggestion
    "symfony/filesystem": "^5.4",
```

---

Thanks for your Comment, i added your suggestion :)